Requirements

```diff
@@ -1,23 +1,23 @@
 Functionele eisen
 
 * Accounts, rollen en rechten:
-    * Gebruikers moeten kunnen inloggen in de app via een beveiligd account
-    * Gebruikers moeten kunnen worden toegevoegd in het systeem
-    * De rollen en rechten per gebruiker (management vs personeelslid) moet kunnen worden toegevoegd, bekeken of aangepast
-    * Gebruikers moeten kunnen worden toegevoegd, bekeken of aangepast
-    * persoonsgegevens (naam, adres, contactgegevens, wachtwoord, functie) van gebruikers moeten kunnen worden toegevoegd, bekeken, aangepast of verwijderd
-    * Afdelingshoofden (management) moeten volledige toegang tot de app kunnen hebben,
-    * Personeelsleden moeten beperkte toegang krijgen tot de app
+    - Gebruikers moeten kunnen inloggen in de app via een beveiligd account
+    - Gebruikers moeten kunnen worden toegevoegd in het systeem
+    - De rollen en rechten per gebruiker (management vs personeelslid) moet kunnen worden toegevoegd, bekeken of aangepast
+    - Gebruikers moeten kunnen worden toegevoegd, bekeken of aangepast
+    - persoonsgegevens (naam, adres, contactgegevens, wachtwoord, functie) van gebruikers moeten kunnen worden toegevoegd, bekeken, aangepast of verwijderd
+    - Afdelingshoofden (management) moeten volledige toegang tot de app kunnen hebben,
+    - Personeelsleden moeten beperkte toegang krijgen tot de app
```

Comment:
Deze twee slaan ook weer op security, maar het zijn niet echt technische eisen, of wel? Ik denk dat dit beter in een ander soort contract of SLA kan worden gezet.